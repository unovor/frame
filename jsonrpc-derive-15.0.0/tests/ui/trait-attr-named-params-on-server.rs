use jsonrpc_derive::rpc;

#[rpc(server, params = "named")]
pub trait Rpc {
}

fn main() {}
