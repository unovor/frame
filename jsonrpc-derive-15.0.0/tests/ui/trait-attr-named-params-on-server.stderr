error: custom attribute panicked
 --> $DIR/trait-attr-named-params-on-server.rs:3:1
  |
3 | #[rpc(server, params = "named")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Server code generation only supports `params = "positional"` (default) or `params = "raw" at this time.
